src/listeners/log-to-console.js

```diff
@@ -23,9 +23,9 @@ export default function(cfg = {}) {
     }
 
     let now = moment(entry._time.stamp).utcOffset(entry._time.utc_offset).toISOString();
-    let levelName = logger.levelToLevelName(entry._level);
+    let levelName = logger.levels.levelToLevelName(entry._level);
```

Comment:
if the intention is to allow the levelToLevelName to be overwritten, someone can extend the class